### ISSUE TITLE
train dataset is OK?

### ISSUE THREAD

**Issue by sukerlove** [yusuketomoto/chainer-fast-neuralstyle#52]
I get http://msvocds.blob.core.windows.net/coco2014/train2014.zip
unzip train2014.zip
then 

suker@suker:~/chainer-fast-neuralstyle$ python train.py -s rio.jpg -d ./train2014 -g 0
/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py:1298: UserWarning: /home/gtx1080/.python-eggs is writable by group/others and vulnerable to attack when used with get_resource_filename. Consider a more secure location (set with .set_extraction_path or the PYTHON_EGG_CACHE environment variable).
  warnings.warn(msg, UserWarning)
num traning images: 82783
82783 iterations, 2 epochs
/usr/local/lib/python2.7/dist-packages/chainer-1.14.0-py2.7-linux-x86_64.egg/chainer/cuda.py:87: UserWarning: cuDNN is not enabled.
Please reinstall chainer after you install cudnn
(see https://github.com/pfnet/chainer#installation).
  'cuDNN is not enabled.\n'
epoch 0
(epoch 0) batch 0/82783... training loss is...235437792.0
(epoch 0) batch 1/82783... training loss is...329452736.0
(epoch 0) batch 2/82783... training loss is...154218272.0
(epoch 0) batch 3/82783... training loss is...63537960.0
(epoch 0) batch 4/82783... training loss is...58619268.0
(epoch 0) batch 5/82783... training loss is...55183628.0
(epoch 0) batch 6/82783... training loss is...54082024.0

is operate OK?
thanks

**Comment by nandexsp** [comment]
sure!

**Comment by sukerlove** [comment]
@nandexsp thanks
but i train
rio_0.model
rio_0.state

rio_1.model
rio_1.state

rio.model
rio.state

I use rio.model, to gen picture, it seems not ok, is any step i'm wrong?
